src/components/Stratosphere/RedirectBanner.tsx

```diff
@@ -5,17 +5,26 @@ import { useLocation, useNavigate } from 'react-router-dom';
 import { ReduxState } from '../../redux/store';
 
 // TODO: Figure out what param chrome should expect
-export const STRATOSPHERE_BANNER_NAME = 'from-aws';
+export const AWS_BANNER_NAME = 'from-aws';
+export const AZURE_BANNER_NAME = 'from-azure';
 
 const RedirectBanner = () => {
   const { pathname, search, hash, state } = useLocation();
   const navigate = useNavigate();
   const params = new URLSearchParams(search);
+  const partner = () => {
```

Comment:
Can we make `partner` directly into a variable instead of a function? Its just an extra step that is not really necessary is the function is called in each render anyways.

You can also wrap it in `useMemo` and use the `search` as its trigger.